director-certs.html.md.erb

```diff
@@ -56,18 +67,70 @@ EOL
   rm ./openssl-exts.conf
 }
 
-generateCert director 10.244.4.2 # <--- Replace with public Director IP
-generateCert uaa-web 10.244.4.2  # <--- Replace with public Director IP
-generateCert uaa-sp 10.244.4.2   # <--- Replace with public Director IP
+generateCert director ${ip}
+generateCert uaa-web ${ip}
+generateCert uaa-sp ${ip}
+
+echo "Generating spruce template..."
+certs=`dirname $0`/certs/${ip}
+
+function indent() {
+  c='s/^/        /'
+  case $(uname) in
+    Darwin) sed -l "$c";;
+    *)      sed -u "$c";;
+  esac
+}
+
+
+cat > certs-${ip}.yml <<EOF
+---
+properties:
+  director:
+    ssl:
+      key: |
+$(cat director.key | indent)
```

Comment:
Instead of having a hierarchy (`director.ssl.key`), those should probably be flat values so they can be merged in by the new golang-cli?